.gitattributes

```diff
@@ -0,0 +1 @@
+*.jpg filter=lfs diff=lfs merge=lfs -text
```

Comment:
I might just take this out because this was meant for uploading the large images I was working with at the time. Since we're merging with this, I don't think we need git lfs anymore here.

Reply:
We prob want to gitignore any test images anyway; we don't want them in this public repo or getting wrapped up in our docker.

Reply:
Yup true. Will add.